### PR TITLE
Added hotjar to application

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
## Notion card
https://www.notion.so/saeloun/6afc9500b2cc42af86be55f37894fb09?v=185bd45b9dc24dfcad9c57c07d32a7c0&p=8b3eaea3fbac4e7ab9c4057a90178419

## Summary
Added hotjar

## Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,19 @@
     <%#= stylesheet_link_tag "inter-font", "data-turbo-track": "reload" %>
     <%= stylesheet_pack_tag "application" %>
     <%= javascript_pack_tag "application"%>
+    <!-- Hotjar Tracking Code for Miru -->
+    <% if Rails.env.production? %>
+      <script>
+        (function(h,o,t,j,a,r){
+          h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+          h._hjSettings={hjid:3014298,hjsv:6};
```
Comment (akhilgkrishnan) [inline app/views/layouts/application.html.erb:16]: can we move the hotjar id to an env?

Comment (sudeeptarlekar) [reply app/views/layouts/application.html.erb:16]: May I know why?

Comment (sudeeptarlekar) [reply app/views/layouts/application.html.erb:16]: Even if you add it in ENV, you can inspect in browser and ID will open in public. 

Even if you consider a case for Google Analytics(GA), which suggests to use browser extension for debugging. If I install that extension and load a site which uses GA, I can get tracking/measurement ID e.g. Opera, Notion

Comment (akhilgkrishnan) [reply app/views/layouts/application.html.erb:16]: Ohh okay.

**Pull request closed without merging**